hack/common.sh

```diff
@@ -356,17 +367,26 @@ function os::build::export_targets() {
   done
 
   if [[ ${#targets[@]} -eq 0 ]]; then
-    targets=("${OS_ALL_TARGETS[@]}")
+    if [[ "${platform}" == linux/amd64 ]]; then
+      targets=("${OS_ALL_TARGETS[@]}" "${OS_SDN_COMPILE_TARGETS_LINUX_AMD64[@]}")
```

Comment:
@smarterclayton the logic is now in hack/build-go.sh, which is the only place that passes an empty targets list to os::build::build_binaries().  All other users correctly pass either OS_CROSS_COMPILE_TARGETS or OS_IMAGE_COMPILE_TARGETS.

build-go.sh gets used from a ton of places that also need the SDN components built, so it now gets the logic.